src/lib/relay/routes.tsx

```diff
@@ -18,6 +18,24 @@ class Artist extends Relay.Route {
   static routeName = "ArtistRoute"
 }
 
+class Conversation extends Relay.Route {
+  static queries = {
+    me: (component, params) => Relay.QL`
```

Comment:
@alloy not sure if you know a way around this - it would be nice to use the existing `MyAccount` route but the Conversation container needs a `conversationID` parameter. I could make it optional on `MyAccount` but that might be worse than just having a unique `Conversation` route. I definitely can't do this:

```graphql
me {
    conversation(id: $conversationID) {
        ${component.getFragment("me", params)}
    }
}
```

because roots can only have one fragment and no fields. Would be nice though.